### PR TITLE
Switch to /admin/workflows/import endpoint to support full app uploads

### PR DESCRIPTION
**Pull request by ghost**
Changing import APIs allows for files to be uploaded to workflows which previously could not be, such as `entity-extensions.json` and `.svg` for logos.

This change adds 1 new dependency, `form-data`, however I believe this package was already a sub-dependency of another package.
If the addition of a new package is not suitable. The useful functionality of it can be replicated without a dependency with minimal effort.

Testing has not been comprehensive. I have successfully uploaded apps, and 1 workflow which has seemed to go smoothly. It may be worth a more comprehensive test before implementation, adding this new API as an optional feature, or allowing the tool to choose between the old and new API based on the content being uploaded.

### CODE REVIEW

File: src/cli/upload.js [hunk 1]
```diff
@@ -44,18 +44,13 @@ module.exports = function(config, workflowDir) {
...
     function updateWorkflow(isCreate) {
-      const content = JSON.stringify({
-        name: isCreate ? workflowName : undefined,
-        base64Content: fs.readFileSync(zip.path).toString('base64')
-      });
+      const form = new FormData();
+      form.append('file', fs.createReadStream(zip.path), {filename:workflowName + '.zip'});
 
-      let message = HttpMessage(resolve(config.host, '/api/admin/workflows/' + (isCreate ? '' : encodeURIComponent(workflowName))));
+      let message = HttpMessage(resolve(config.host, '/api/admin/workflows/import' + (isCreate ? '' : encodeURIComponent(workflowName))));
```
Comment (andrey-skl) [inline src/cli/upload.js]: I believe, we can go even further and use apps endpoint? (It is used in new Apps Admin UI on `admin/apps` URL)

```
POST api/admin/apps/import
```

Worth mentioning that there should be no changed done to path for subsequential Apps uploading: the URL is always the same

Comment (ghost) [reply src/cli/upload.js]: ```POST /api/admin/workflows/import``` is what's used in the current YouTrack UI when importing an app.
If ` POST api/admin/apps/import` also will allow for the import of standard workflows, then that could be used too. However if it ONLY supports uploading apps, then it should either be split into its own command/option, or not used.

Comment (andrey-skl) [reply src/cli/upload.js]: Turns out this apps API is not yet actually released, so we need to stick to Workflow API anyway : )

@TPG-Teklada Anyway we shouldn't change the URL for updating existing workflows I believe, it should be always `/api/admin/workflows/import` without adding workflow name on the end

Comment (ghost) [reply src/cli/upload.js]: @andrey-skl , ahh good point! I'll remove the workflow name and update the PR

**Review by andrey-skl — APPROVED** [review]

**Comment by ghost** [comment]
@andrey-skl, will there be more testing before release to NPM. I can see the contributors file you've added. Should that be performed from JetBrains' side, or me/external contributors?

**Comment by andrey-skl** [comment]
@TPG-Teklada Hi! I've tested it and it works fine, so I've already released as `0.2.1`. We don't run any full-testing cycle here as the change is quite minor

https://www.npmjs.com/package/@jetbrains/youtrack-scripting/v/0.2.1

**Pull request merged**